Dockerfile

```diff
@@ -1,14 +1,17 @@
 FROM rocker/tidyverse:4.2
 MAINTAINER naqvia@chop.edu
 WORKDIR /rocker-build/
+
+ENV BCF_VERSION 1.9
```

Comment:
What is this doing? When load your image, it has version 1.13 - assuming it only installs latest.

>Program: bcftools (Tools for variant calling and manipulating VCFs and BCFs)
Version: 1.13 (using htslib 1.13+ds)

Can you try something like this instead: https://github.com/d3b-center/OpenPedCan-analysis/blob/74f73764ce56b3d580d825563e9b0a8709d734fe/Dockerfile#L69-L74